src/subscript/prtvol2csv/prtvol2csv.py

```diff
@@ -205,6 +205,7 @@ def currently_in_place_from_prt(
     date_matcher_ecl = re.compile(r"^\s{2}REPORT\s{3}0")
     # Flow: Look for "Report step  0" at start of line
     date_matcher_flow = re.compile(r"^Report step\s{2}0")
+    initial_date_found = False
```

Comment:
Related to the test?

Reply:
Not really, I was trying to solve warning from pylance. 😅

Reply:
This is a kind 'may be Unbounded'-warning? Is there a ruff issue that that can be added to similarly require this? (it is a good find, but should probably go in its own commit with a corresponding git message)

Reply:
I can't see to find ruff rule for this. But I agree that it should be in its own commit.